src/main/java/ch/njol/skript/expressions/ExprItemAttributeModifiers.java

```diff
@@ -0,0 +1,167 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.attribute.AttributeModifier;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+
+@Name("Item Attribute Modifiers")
+@Description({
+	"The attribute modifiers of a given attribute type of an item.",
+	"Returns nothing if the item has no attribute modifiers for the specified attribute type.",
+	"One can add, remove, set, or reset the attribute modifiers of an item."
+})
+@Examples({
+	"set {_modifier} to a new attribute modifier named \"Damage\" with value 5 and for the main hand slot",
+	"set {_item} to a diamond sword",
+	"add {_modifier} to the attack damage attribute modifiers of {_item}",
+})
+@Since("INSERT VERSION")
+public class ExprItemAttributeModifiers extends PropertyExpression<ItemType, AttributeModifier> {
+
+	static {
+		register(
+			ExprItemAttributeModifiers.class,
+			AttributeModifier.class,
+			"%attributetype% attribute[s] [modifier[s]]",
```

Comment:
Would being able to get the _all_ attribute modifiers on an item be something useful?

Reply:
Not really, since they hide information on the attribute type they were assigned to.